app/models/mixins/ansible_authentications.rb

```diff
@@ -29,43 +29,43 @@ def ansible_secret_key=(key)
     auth.save!
   end
 
-  def ansible_rabbitmq_password
-    auth = authentication_type(ANSIBLE_RABBITMQ_TYPE)
-    auth.nil? ? nil : auth.password
+  def ansible_rabbitmq_authentication
+    authentication_type(ANSIBLE_RABBITMQ_TYPE)
   end
 
-  def ansible_rabbitmq_password=(password)
-    auth = authentication_for_type(ANSIBLE_RABBITMQ_TYPE, "Ansible Rabbitmq Password")
+  def set_ansible_rabbitmq_authentication(password, userid = "tower")
```

Comment:
I wonder if we should do kwargs here, it seems weird to have the the parameters in this order, only because the userid is not required.